config/dev.exs

```diff
@@ -7,6 +7,6 @@ config :aprb, Aprb.Repo,
   adapter: Ecto.Adapters.Postgres,
   username: System.get_env("DB_USER"),
   password: System.get_env("DB_PASSWORD"),
-  database: System.get_env("DB_NAME"),
-  hostname: System.get_env("DB_HOST"),
+  database: "aprb_dev",
+  hostname: "localhost",
```

Comment:
We may want to make `localhost` default so something like `hostname: System.get_env("DB_HOST") || "localhost"` but honestly this fixes 90% of our cases and once someone started using it in dev we can always play with this more! thanks for this! 💯